app/src/main/java/org/xbmc/kore/jsonrpc/HostConnection.java

```diff
@@ -606,7 +638,7 @@ public void run() {
                 for (final PlayerNotificationsObserver observer :
```

Comment:
Instead I think we can do better by using a `HashMap` to hold each list of observers and using the `NOTIFICATION_NAME`s as keys. This would allow us to do away with the whole if structure.